### PR TITLE
PHPFPM: using file/tail alternative to logging

### PR DESCRIPTION
**Pull request by bryanlatten**
TEMPORARY

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -32,7 +32,6 @@ RUN apt-get update && \
         php5-json \
         php5-xdebug \
         newrelic-php5 \
-        wget \
```
Comment (bryanlatten) [inline Dockerfile]: already present above

File: Dockerfile [hunk 2]
```diff
@@ -49,8 +48,10 @@ RUN pecl install igbinary-1.2.1 && \
 # Prevent newrelic daemon from auto-spawning; uses newrelic run.d script to enable at runtime, when ENV variables are present
 # @see https://docs.newrelic.com/docs/agents/php-agent/advanced-installation/starting-php-daemon-advanced
 RUN sed -i "s/;newrelic.daemon.dont_launch = 0/newrelic.daemon.dont_launch = 3/" /etc/php5/mods-available/newrelic.ini && \
-    sed -i "s/listen = \(.*\)\+/listen = 127.0.0.1:9000/" /etc/php5/fpm/pool.d/www.conf
+    sed -i "s/listen = \(.*\)\+/listen = 127.0.0.1:9000/" /etc/php5/fpm/pool.d/www.conf && \
+    sed -i "s/chdir = \//chdir = \/app/" /etc/php5/fpm/pool.d/www.conf
```
Comment (bryanlatten) [inline Dockerfile]: saw a random posting suggesting that this should always be set

File: Dockerfile [hunk 3]
```diff
@@ -76,8 +77,3 @@ ONBUILD COPY ./ /app/
 # TODO: script needs to be called AFTER downstream build is performed,
 #   ONBUILD instruction gets called BEFORE, so not useful
 # RUN /bin/bash /clean.sh
-
-
-
-EXPOSE 80
```
Comment (bryanlatten) [inline Dockerfile]: these are set in the parent, no need to duplicate

File: Dockerfile (hunk 1 above)
Comment (chrishenry) [inline Dockerfile]: nice cleanup

File: container/root/run.d/99-phpfpm.sh [hunk 4]
```diff
@@ -31,4 +30,11 @@ sed -i "s/;catch_workers_output/catch_workers_output/" $PHPFPM_CONF
 # 5. reconnecting the stdout to current stdout
 # 6. backgrounding that process
 echo '[php-fpm] starting (background)'
-php5-fpm -F -O 2>&1 | sed -u 's,.*: \"\(.*\)$,\1,'| sed -u 's,"$,,' 1>&1 &
+
+# TODO: temporarily leverage normal logfile + startup mechanism,
+#  to determine if logging redirection prevents segfaults from being surfaced
+service php5-fpm start
+tail -f /var/log/php5-fpm.log | sed -u 's,.*: \"\(.*\)$,\1,'| sed -u 's,"$,,' 1>&1 &
```
Comment (chrishenry) [inline container/root/run.d/99-phpfpm.sh]: awesome, there's that missing log.

**Comment by chrishenry** [comment]
:+1:

**Pull request merged**